frontend/src/components/Forms/reports/AddReportForm.js

```diff
@@ -214,6 +236,8 @@ export default compose(
     mineReportDefinitionOptions: getMineReportDefinitionOptions(state),
     selectedMineReportCategory: selector(state, "mine_report_category"),
     selectedMineReportDefinition: selector(state, "mine_report_definition_guid"),
+    currentDueDate: selector(state, "due_date"),
+    formMeta: state.form.ADD_REPORT,
```

Comment:
Trying to find a better way of getting the form meta. but i never got this to work. 

https://redux-form.com/7.0.2/docs/api/selectors.md/#-code-getformmeta-formname-string-code-returns-code-state-gt-formmeta-object-code-